{"name": "Вход курьера с неверным  паролем", "status": "passed", "description": "Ожидаемый результат не совпадает с фактическим. Помимо сообщения выводится код отввета 404", "start": 1724601128287, "stop": 1724601128612, "uuid": "42771e32-7d71-4c94-bfc7-f74bde9d8abc", "historyId": "e02c35e245704ecc13d1413cb8ad9170", "testCaseId": "e02c35e245704ecc13d1413cb8ad9170", "fullName": "tests.test_api_sign_in.test_api_sign_in_post.TestApiSignInPost#test_sign_in_courier_wrong_password", "labels": [{"name": "parentSuite", "value": "tests.test_api_sign_in"}, {"name": "suite", "value": "test_api_sign_in_post"}, {"name": "subSuite", "value": "TestApiSignInPost"}, {"name": "host", "value": "MacBook-Pro-2.local"}, {"name": "thread", "value": "64895-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api_sign_in.test_api_sign_in_post"}]}